### ISSUE TITLE
[?] is there a way to access an Android Phone with Broken Display ?

### ISSUE THREAD

**Issue by ThePredators** [Genymobile/scrcpy#3712]
Hello, 

I'm just wondering if there is a way to access my android phone with `scrcpy` to back up my data before sending it for fixing. I was thinking if there some sort of command to unlock it with my pin, and allow usb debuging so that i can mirror it on my macOS and start the backup. 

Any Help on this is extremely appreciated. 

Thanks a lot.

**Comment by ThePredators** [comment]
I tried both stuff given in these tickets without success :/ 

https://github.com/Genymobile/scrcpy/issues/71
https://github.com/Genymobile/scrcpy/issues/354

**Comment by rom1v** [comment]
Was USB debugging enabled on the device (even if the computer is not authorized)?

**Comment by Sam-LearnsToCode** [comment]
Hey I have the same issue but I use a pattern on my device, It is authorized and no timeout is also enabled in dev options 
@rom1v can you help ?

**Comment by rom1v** [comment]
If it's authorized, you can probably disable the lock with:

```
adb shell cmd lock_settings clear --old <CREDENTIAL>
```

If you use a pattern as credential, the dots are numbered as follow:
```
 1 2 3
 4 5 6
 7 8 9
```

For example, if your pattern is a 'L':
```
adb shell cmd lock_settings clear --old 14789
```

See https://nerdschalk.com/change-remove-lockscreen-pattern-pin-password-adb-android-oreo/

**Comment by ThePredators** [comment]
Thanks guys a lot for your answers. 

Mean while i removed the back cover of my Android phone and played a little bit with display and managed to see half screen and get touch pad back working. 

I found an easy and very useful solution to my problem. I just plugged a keyboard, mouse "this replace touch", and i was able to set the password and get to the phone. Then all i had to do is connect an HDMI cable an accept screen mirroring. 

All ended well i could backup 100% of my data :)
